sr/comp/mixtape/cli.py

```diff
@@ -20,15 +20,14 @@
     datefmt='%Y-%m-%dT%H:%M:%S',
 )
 
-
```

Comment:
Looks like the linter doesn't like this line being removed:

Reply:
Interesting.... the linter works in mysterious ways 😆 

Changed in f0bbf4ff236f8fa080705ee424786fa7cf6e365f.